tests/integration/buildcmd/build_integ_base.py

```diff
@@ -396,3 +396,145 @@ def _verify_invoke_built_functions(self, template_path, functions, error_message
+
+        npm_projects = ["cdk-example-rest-api-gateway"]
+
+        for project in npm_projects:
+            project_path = self.get_project_path(project)
+            node_modules_path = os.path.join(project_path, "node_modules")
+            package_lock_path = os.path.join(project_path, "package-lock.json")
+            if os.path.exists(node_modules_path):
+                shutil.rmtree(node_modules_path, ignore_errors=True)
+            if os.path.exists(package_lock_path):
+                os.remove(package_lock_path)
+
+    def copy_source_to_temp(self, project_name):
+        source = self.get_project_path(project_name)
+        destination = str(Path(self.working_dir, "source"))
+        if not os.path.exists(destination):
+            self.temp_source = shutil.copytree(source, destination)
+
+    def get_command_list(
+        self,
+        build_dir=None,
+        base_dir=None,
+        manifest_path=None,
+        use_container=None,
+        parameter_overrides=None,
+        mode=None,
+        function_identifier=None,
+        debug=False,
+        cached=False,
+        cache_dir=None,
+        parallel=False,
+        container_env_var=None,
+        container_env_var_file=None,
+        build_image=None,
+        project_type=None,
+    ):
+
+        command_list = [self.cmd, "build", "-b", str(self.default_build_dir)]
+
+        if parameter_overrides:
+            command_list += ["--parameter-overrides", self._make_parameter_override_arg(parameter_overrides)]
+
+        if build_dir:
+            command_list += ["-b", build_dir]
+
+        if base_dir:
+            command_list += ["-s", base_dir]
+
+        if manifest_path:
+            command_list += ["-m", manifest_path]
+
+        if use_container:
+            command_list += ["--use-container"]
+
+        if debug:
+            command_list += ["--debug"]
+
+        if cached:
+            command_list += ["--cached"]
+
+        if cache_dir:
+            command_list += ["-cd", cache_dir]
+
+        if parallel:
+            command_list += ["--parallel"]
+
+        if container_env_var:
+            command_list += ["--container-env-var", container_env_var]
+
+        if container_env_var_file:
+            command_list += ["--container-env-var-file", container_env_var_file]
+
+        if build_image:
+            command_list += ["--build-image", build_image]
+
+        if project_type:
+            command_list += ["--project-type", project_type]
+
+        return command_list
+
+    def get_project_path(self, project_name):
+        return str(Path(self.test_data_path, project_name))
+
+    def verify_build_success(self, project_name, cmd=None):
+        self.copy_source_to_temp(project_name)
+
+        if cmd:
+            cmd_list = cmd
+        else:
+            cmd_list = self.get_command_list()
+
+        LOG.info("Running Command: {}".format(cmd_list))
+        LOG.info(cmd_list)
```

Comment:
It seems like a repeated logging of `cmd_list`?

Reply:
Will remove this.